app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable intended to track all living instances of `Animal`. Ensure that all instances are properly removed from this list when they 'die' to prevent memory leaks or incorrect state tracking.